src/h5web/toolbar/toolbars.ts

```diff
@@ -0,0 +1,5 @@
+export { default as LineToolbar } from './LineToolbar';
+export { default as HeatmapToolbar } from './HeatmapToolbar';
+export { default as ComplexToolbar } from './ComplexToolbar';
+export { default as ComplexLineToolbar } from './ComplexLineToolbar';
+export { default as RgbToolbar } from './RgbToolbar';
```

Comment:
Creating a couple of "index" files to clean up the imports in the vis-packs' vis definitions files.